torchci/components/NavBar.module.css

```diff
@@ -7,6 +7,7 @@
   width: 100%;
   min-height: 60px;
   align-items: center;
+  transition: background 0.3s ease, box-shadow 0.3s ease;
```

Comment:
My .css foo is pretty stale, what browser will support those?

Reply:
https://caniuse.com/css-transitions -> all of them except for ie9 and opera mini